Tasks/scss-targets.js

```diff
@@ -30,12 +32,40 @@ function compile(source, destination, compatibleBrowsers) {
             ],
         }))
         .pipe(sourcemaps.init({ loadMaps: true }))
-        .pipe(sass({ linefeed: 'crlf' }).on('error', sass.logError))
+        .pipe(sass({ linefeed: process.platform === 'win32' ? 'crlf' : 'lf' }).on('error', sass.logError))
         .pipe(postcss([autoprefixer({ overrideBrowserslist: compileCompatibleBrowsers })]))
         .pipe(sourcemaps.write('.', { includeContent: true }))
+        .pipe(normalizeSourceMap())
         .pipe(gulp.dest(compileDestination));
 }
 
+function normalizeSourceMap() {
+    const normalize = (buffer) => {
```

Comment:
Why don't you define a plain old `function` here? What are your reasons to you use a named lambda expression?

Reply:
I don't know. This was the result of some frantic experimentation so I didn't pay enough attention to uniform style.
I have ran the file through ESLint now and updated the code. (also moved the whole method up because ESLint doesn't like when we use [hoisting](https://developer.mozilla.org/en-US/docs/Glossary/Hoisting), which is understandable)

Reply:
Great! Thanks for the details.